jdrfandroidbleparser/src/main/java/org/ehealthinnovation/jdrfandroidbleparser/bgm/characteristic/BaseCharacteristic.kt

```diff
@@ -57,7 +75,8 @@ abstract class BaseCharacteristic(characteristic: BluetoothGattCharacteristic?,
             c.getStringValue(offset)?.let {
                 offset += it.toByteArray().size
                 return it
-            } ?: throw NullPointerException("Offset \"$offset\" does not relate to valid String value...")
+            }
+                    ?: throw NullPointerException("Offset \"$offset\" does not relate to valid String value...")
```

Comment:
Is this one from autoformat? It looks like detached from the previous clause.

Reply:
yeah, the autoformatter didn't like that. Fixed.